lib/zoom/utils.rb

```diff
@@ -34,6 +34,13 @@ def parse_response(http_response)
       def extract_options!(array)
         params = array.last.is_a?(::Hash) ? array.pop : {}
         process_datetime_params!(params)
+        convert_param_names!(params)
+      end
+
+      def convert_param_names!(params)
```

Comment:
This creates tight coupling between this module and `lib/zoom/actions/token.rb`. While I like these new names better as they match the API parameters exactly, I don't think that this is the right place to put this backward compatibility logic.